Kulyk_Lab/app/src/main/java/com/nazar/kulyk_lab/netconnection/RetrofitClient.java

```diff
@@ -0,0 +1,32 @@
+package com.nazar.kulyk_lab.netconnection;
+
+import android.app.Application;
+
+import com.nazar.kulyk_lab.interfaces.RijksmuseumApi;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitClient extends Application{
```

Comment:
Your application is not a RetrofitClient. Other things, that have nothing to do with retrofit will be stored in here. Correct approach is to have something like

```
public class ApplicationEx extends Application {
...
   public ApiClient apiClient() {
      ...
   }
```